includes/class-wc-payments-customer-service.php

```diff
@@ -236,4 +236,11 @@ private function recreate_customer( $user, $name, $email ) {
 
 		return $this->create_customer_for_user( $user, $name, $email );
 	}
+
+	/**
+	 * Returns the name of the customer option meta, taking test mode into account.
+	 */
+	private function get_customer_id_option() {
+		return self::WCPAY_CUSTOMER_ID_OPTION . ( WC_Payments::get_gateway()->is_in_test_mode() ? '_test' : '' );
```

Comment:
This could be simpler./ 
```suggestion
if ( WC_Payments::get_gateway()->is_in_test_mode() ) {
  return self::WCPAY_CUSTOMER_ID_OPTION . '_test' ;
} eslse {
 return  self::WCPAY_CUSTOMER_ID_OPTION;
}
```

Reply:
Thanks for catching that. I've simplified it in https://github.com/Automattic/woocommerce-payments/pull/918/commits/4506fd3a0510488b01483366029ff360d04adf49.